cli/explain/compactgraph/testdata/generate.go

```diff
@@ -68,28 +69,42 @@ public class App {
 `
 
 func main() {
+	// bazelisk prepends the current Bazel binary to the PATH, but we want "bazel" to reference bazelisk itself.
```

Comment:
I sent https://github.com/bazelbuild/bazelisk/pull/612 to make this simpler in the future.